gulpfile.babel.js

```diff
@@ -168,13 +169,6 @@ export function styles() {
 
 	return gulp.src(paths.styles.src)
 	.pipe(print())
-	.pipe(phpcs({
-		bin: 'vendor/bin/phpcs',
-		standard: 'WordPress',
-		warningSeverity: 0
-	}))
-	// Log all problems that was found
-	.pipe(phpcs.reporter('log'))
```

Comment:
This shouldn't be removed @ai

Reply:
Oops. I thought it is a way to run CSS Lint